README.md

```diff
@@ -1 +1,3 @@
-# todo-list-codereview
\ No newline at end of file
+# todo-list-codereview
+
+This project has a copy from the code created for the To Do List project. The purpose is to practice code review
```

Comment:
- You might want to consider updating your README file because it provides an introduction to the files contained in the repository which gives anyone using your repository a better understanding of your project.